TundraTD/Assets/Scripts/Camera/CameraMotion.cs

```diff
@@ -0,0 +1,201 @@
+﻿using System;
+using UnityEngine;
+
+// У нас по-настоящему инклюзивная, гендерно нейтральная команда!
+public class CameraMotion : MonoBehaviour
+{
+    private Camera Cam;
+    private bool Flag;
+    private float XPos;
+    private float ZPos;
+    private Vector2 ChangePos1;
+    private Vector2 ChangePos2;
+    private Touch Touch;
+    private Touch Touch1;
+    private Touch Touch2;
+    private float LastPositionDistance;
+    private float Zoom;
```

Comment:
Follow coding conventions: class fields should be called with underscore and lowercase like _variable